openrpc/src/methods/getTransactions.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "getTransactions",
+  "summary": "returns a list of transactions with their details",
+  "description": "The getTransactions method returns details of a list of transactions starting from the user specified start ledger or cursor. The RPC system maintains a restricted history of recently processed transactions, with the default retention window set at 1440 ledgers, approximately equivalent to a 2-hour timeframe. For private RPC instances, it is possible to modify the retention window value by adjusting the transaction-retention-window configuration setting. Due to this limited history, clients should make sure that the requests for transactions fall within the ledger range stored by their rpc instance.",
```

Let's rephrase this to be more generic and read more like documentation rather than an operator guide, and especially not distinguishing between public/private instances (since we don't run one).

```suggestion
  "description": "The `getTransactions` method return a detailed list of transactions starting from the user specified starting point that you can paginate as long as the pages fall within the history retention of their corresponding RPC provider.",
```

(Just a suggestion, feel free to use your own words.)

---

No I agree. My default assumption is that these are docs for someone running their own instances. However, we should also cater to those who are depending on infra operators.